mod_customized/controllers.py

```diff
@@ -78,7 +78,9 @@ def index():
             )
             # Show error if github fails to recognize commit
             response = requests.get(api_url)
-            if response.status_code == 404:
+            if response.status_code == 500:
```

Comment:
This is not a big improvement. Why not have it with the other non- codes?

Reply:
This was for Server Errors in Case GitHub is having trouble